### PR TITLE
Reactivate Landingpage and tests

### PR DESCRIPTION
**Pull request by sbrunner**
See also: https://github.com/qgis/QGIS/commit/3eb888457c5169d0b66aaf85193e0351016c17a2
related to: GSGMF-1472

### CODE REVIEW

File: acceptance_tests/conftest.py [hunk 1]
```diff
@@ -23,3 +23,11 @@ def connection(wait):
     Fixture that returns a connection to a running batch container.
     """
     return Connection("http://qgis:8080/", "http://www.example.com/")
+
+
+@pytest.fixture
+def connection_landingpage(wait):
```
Comment (arnaud-morvan) [inline acceptance_tests/conftest.py:29]: Note that it seems we could add `autouse=True` on fixture `wait`

File: acceptance_tests/docker-compose.yaml [hunk 2]
```diff
@@ -17,6 +17,23 @@ services:
       - acceptance_config
     user: www-data
 
+  qgis-landingage:
```
Comment (arnaud-morvan) [inline acceptance_tests/docker-compose.yaml:20]:
```suggestion
  qgis-landingpage:
```

File: acceptance_tests/docker-compose.yaml [hunk 3]
```diff
@@ -17,6 +17,23 @@ services:
...
+    image: camptocamp/qgis-server:${DOCKER_TAG}
+    environment:
+      QGIS_SERVER_LOG_LEVEL: '0'
+      QGIS_CATCH_SEGV: '1'
+      QGIS_AUTH_DB_DIR_PATH: /etc/qgisserver/
+      PGSERVICEFILE: /etc/qgisserver/pg_service.conf
+      QGIS_SERVER_LANDING_PAGE_PREFIX: /mapserv_proxy/qgis
+      QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES: /etc/qgisserver
```
Comment (arnaud-morvan) [inline acceptance_tests/docker-compose.yaml:28]: Do we really need a new service for these 2 additional env vars ?

File: acceptance_tests/conftest.py (hunk 1 above)
Comment (arnaud-morvan) [reply acceptance_tests/conftest.py:29]: Also note that it waits for `qgis`, not for `qgis-landingpage`.

File: acceptance_tests/docker-compose.yaml (hunk 3 above)
Comment (arnaud-morvan) [reply acceptance_tests/docker-compose.yaml:28]: May be yes because qgis has a QGIS_PROJECT_FILE

Comment (sbrunner) [reply acceptance_tests/docker-compose.yaml:29]: Comment added :-)

**Review by arnaud-morvan — APPROVED** [review]

**Pull request merged**